exercises/say/canonical-data.json

```diff
@@ -92,7 +100,7 @@
       "input": {
         "number": 1002345
       },
-      "expected": "one million two thousand three hundred forty-five"
+      "expected": "one million twenty-three thousand forty-five"
```

Comment:
you would want to change this back, if leaving 1002345 untouched.

Reply:
Whoops.